applications/stative_network.md

```diff
@@ -0,0 +1,312 @@
+|    0b. | Documentation                 | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works.                                                                                                                                                           |
+|    0c. | Testing and Testing Guide     | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                                                                                                                                                                                         |
+|    0d. | Docker                        | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone.                                                                                                                                                                                                                                                                                              |
+|    0e. | Article                       | We will publish an **article** on our bloq showcasing the achievements created in this grand application. Further, this includes snippets from the documentation to give interested reader a good starting point.                                                                                                                                                                                       |
+|     1. | Dockerfile (Base Environment) | Setup & configure a basic Substrate environment w/ final consensus & governance rules. This step includes the basic Dockerfiles, which we will extend and use the other steps, to spin up a local two node system.                                                                                                                                                                                      |
+|     2. | Substrate module: Quote       | Connect Chainlinks parachain to retrieve oracle prices for the NSTs substrate module (MS2.1) as well as the NGT substrate module (MS3.1) using an implementation of Chainlinks substrate module. This module will digest raw pricing data and calculate normalized weighted pricing data. The final deliverable here is a Substrate module (incl. updated `src`, Dockerfile, documentation & tutorial). |
```

Comment:
What do you mean by Connect Chainlinks parachain? It's probably better to focus only on the integration of the chainlink module on a local chain.  Btw. did you take a look at the code by Acala, and how will your solution be different regarding the use of oracle data?

Reply:
Could you still remove the connection to the chainlink parachain from the delivery here? (There is no chainlink parachain)